scripts/core/PROG.rst

```diff
@@ -611,7 +611,7 @@ External memory handles may be imported from other APIs, or exported for use in
 Importing and exporting external memory is an optional feature.
 Devices may describe the types of external memory handles they support using ${x}DeviceGetExternalMemoryProperties.
 
-Importing and exporting external memory is supported for device memory allocations and images.
+Importing and exporting external memory is supported for device and host memory allocations and images.
```

Comment:
```suggestion
%if ver >= 1.5:
Importing and exporting external memory is supported for device and host memory allocations and images.
%else:
Importing and exporting external memory is supported for device memory allocations and images.
%endif
```

I think we'll want version checks here because we only supported device allocations pre-v1.5.

Reply:
@bashbaug  you are correct. Added.